packages/tools/src/tools/SculptorTool/ISculptToolShape.ts

```diff
@@ -0,0 +1,61 @@
+import type { Types } from '@cornerstonejs/core';
+import { SVGDrawingHelper, EventTypes, Annotation } from '../../types';
+import { PushedHandles } from './CircleSculptCursor';
+import { SculptData } from '../SculptorTool';
+
+/**
+ * This interface defines a contract for implementing various shapes within sculptor tool.
+ * Classes such as `CircleSculptCursor` adhere to this interface, providing specific implementations for sculptor tools to utilize circle shapes during sculpting operations.
+ */
```

Comment:
The constructor for this would look like:

<T extends IScultToolShape>registerShape(name: string, toolClass: new () => T)